apps/judicial-system/web/src/components/RulingInput/RulingInput.tsx

```diff
@@ -14,41 +14,32 @@ import {
 interface Props {
   workingCase: Case
   setWorkingCase: React.Dispatch<React.SetStateAction<Case>>
-  isCaseUpToDate: boolean
   isRequired: boolean
   rows?: number
 }
 
 const RulingInput: React.FC<Props> = (props) => {
-  const {
-    workingCase,
-    setWorkingCase,
-    isCaseUpToDate,
-    isRequired,
-    rows,
-  } = props
+  const { workingCase, setWorkingCase, isRequired, rows } = props
   const { updateCase, autofill } = useCase()
   const { formatMessage } = useIntl()
   const [rulingErrorMessage, setRulingErrorMessage] = useState('')
 
   useEffect(() => {
-    if (isCaseUpToDate) {
```

Comment:
isCaseUpToDate is set to false after one sec. Therefore, it is never set to `true` in modals.